src/config.c

```diff
@@ -60,8 +60,8 @@ static FILE* open_file(void)
     size_t len;
     const char* postfix = "/swayimg/config";
     const size_t postfix_len = strlen(postfix);
+    char* config_dir = getenv("XDG_CONFIG_HOME");
```

Comment:
You don't need to remove `const`.

Reply:
Ah, right, `const` in C always confuses me.